pycm/pycm_util.py

```diff
@@ -62,9 +62,9 @@ def class_filter(classes, class_name):
     """
     Filter classes by comparing two lists.
 
-    :param classes: matrix classes
+    :param classes: confusion matrix classes
     :type classes: list
-    :param class_name: sub set of classes
+    :param class_name: subset of classes list
```

Comment:
`sub set` is better as you used in other files (almost 10 times `sub set` but not here)